signer/src/emily_client.rs

```diff
@@ -82,6 +101,44 @@ impl EmilyInteract for EmilyClient {
         //     .await
         //     .map_err(EmilyClientError::GetDeposits)?;
     }
+
+    async fn update_broadcasted_deposits<'a>(
+        &'a self,
+        transaction: &'a UnsignedTransaction<'a>,
+        bitcoin_chain_tip: &'a BitcoinBlockRef,
+    ) -> Result<UpdateDepositsResponse, Error> {
+        let deposits = transaction.requests.iter().filter_map(|e| match e {
+            RequestRef::Deposit(d) => Some(d),
+            _ => None,
+        });
+
+        let mut update_request = Vec::new();
+        for deposit in deposits {
+            update_request.push(DepositUpdate {
+                bitcoin_tx_output_index: i32::try_from(deposit.outpoint.vout)
+                    .map_err(|_| Error::TypeConversion)?,
```

Comment:
Hmm, this doesn't seem right (no ones fault, so definitely not yours). This seems unnecessary and can be fixed by modifying how the source gets generated. 
1. In the `.generated-sources/emily/openapi/package.json` filed, add `--additional-properties=preferUnsignedInt=true` to the end of the `build-rust` "script" command on line 4.
2. Then do `make build`.

These signed integer fields should now be unsigned integers.